### DISCUSSION TITLE
Effective Posting

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/fds#12448] [Announcements]
Please read this article on [Effective Posting](https://github.com/firemodels/fds/wiki/Effective-Posting).

**Comment by ericvmueller** [comment]
Ive seen other repos make use of 'templates' for issue posting. So every time before someone opens an issue they are faced with text reminding them of etiquette, guides, searching issue history, and also asking for basic info (version number, etc). 

It looks like there's a similar feature for Discussions ...something to consider perhaps
https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms

**Reply by rmcdermo** [reply]
OK, I added a couple of templates.  Have a look and edit as you like.
